hydromt_wflow/wflow.py

```diff
@@ -1867,7 +1867,10 @@ def write_forcing(
             missings = False
             for da in self.forcing.values():
                 if "time" in da.coords:
-                    times = da.time.values
+                    if not isinstance(da.indexes["time"], pd.DatetimeIndex):
```

Comment:
Based on the xarray docs it seems that to_datetimeindex is a method of the `CFTimeIndex` object only. To avoid applying it to an time object type that does not have this method it would suggest to change it into:
`if isinstance(da["time"], xr.CFTimeIndex):`
or more general
`if hasattr(da['time'], 'to_datetimeindex'):`